judge/models/runtime.py

```diff
@@ -147,6 +149,13 @@ def disconnect(self, force=False):
 
     disconnect.alters_data = True
 
+    def toggle_disabled(self):
+        self.is_disabled = not self.is_disabled
+        self.save(update_fields=['is_disabled'])
```

Comment:
I think this `save` should come after the RPC, so we don't end up (in case of an error) with a bridge that does not agree in state with the site.